package.json

```diff
@@ -153,11 +153,13 @@
   "version": "4.0.0",
   "dependencies": {
     "@popperjs/core": "^2.7.2",
+    "@reach/dialog": "^0.13.2",
     "@reach/menu-button": "^0.12.1",
     "classcat": "^4.1.0",
-    "focus-trap-react": "^8.3.2",
+    "framer-motion": "^3.10.0",
```

Comment:
Couple of things:

* The api for css transition is pretty outdated and not so great.
*  Ryan Florence (creator of Reach UI) refused to support it for their dialog. So we can't use it even if we wanted to.
* Couple of libraries for animation available. react-spring is nice but has memory leak issues, and many months ago I tried this same PR with react-spring and animations were not completing in staging.
* framer-motion is probably the most accessible, up to date, maintained, and powerful animation library, so this is what I'm opting to use here.